sql/views.sql

```diff
@@ -91,179 +106,190 @@ FROM _timescaledb_config.bgw_job j
   LEFT JOIN _timescaledb_internal.bgw_job_stat js ON js.job_id = j.id;
 
 -- views for continuous aggregate queries ---
-CREATE OR REPLACE VIEW timescaledb_information.continuous_aggregates as
-  SELECT
-    cagg.user_view_schema AS view_schema,
-    cagg.user_view_name AS view_name,
-    viewinfo.viewowner as view_owner,
-    bgwjob.schedule_interval,
-    cagg.materialized_only,
-    ht.schema_name as materialization_hypertable_schema,
-    ht.table_name as materialization_hypertable_name,
-    directview.viewdefinition as view_definition
-  FROM  _timescaledb_catalog.continuous_agg cagg
-        LEFT JOIN _timescaledb_config.bgw_job bgwjob ON bgwjob.hypertable_id = cagg.mat_hypertable_id,
-        _timescaledb_catalog.hypertable ht, LATERAL
-        ( select C.oid, pg_get_userbyid( C.relowner) as viewowner
-          FROM pg_class C LEFT JOIN pg_namespace N on (N.oid = C.relnamespace)
-          where C.relkind = 'v' and C.relname = cagg.user_view_name
-          and N.nspname = cagg.user_view_schema ) viewinfo, LATERAL
-        ( select pg_get_viewdef(C.oid) as viewdefinition
-          FROM pg_class C LEFT JOIN pg_namespace N on (N.oid = C.relnamespace)
-          where C.relkind = 'v' and C.relname = cagg.direct_view_name
-          and N.nspname = cagg.direct_view_schema ) directview
-  WHERE cagg.mat_hypertable_id = ht.id;
+CREATE OR REPLACE VIEW timescaledb_information.continuous_aggregates AS
+SELECT cagg.user_view_schema AS view_schema,
+  cagg.user_view_name AS view_name,
+  viewinfo.viewowner AS view_owner,
```

Comment:
Non-formatting nit. I guess we used just `owner` for the `hypertables` and `data_nodes` views. Perhaps we should adopt the same here?